doc/UsersGuide/BackgroundInfo/CCPPUpdates.rst

```diff
@@ -3,92 +3,59 @@
 .. _CCPPUpdates:
 
 ================================================
-CCPP Updates for the SRW App v2.2.0 Release
+CCPP Updates for the SRW App v3.0.0 Release
```

Comment:
@MichaelLueken Please make sure @jdkublnick or @selbigmtnwx23 update this information once it is received from @mkavulich . Can be done in a different PR.